playbooks/roles/register_host/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+dnsname_or_ip: "{{ hostvars.bastion.ansible_host }}"
+hostname: "scenario_host"
```

Comment:
Maybe should be `server`, or should not be overridden in each playbook?

Reply:
ok